docs/images.md

```diff
@@ -115,7 +118,7 @@ metadata:
 spec:
   containers:
     - name: foo
-      image: registry.example.com/bar/fo
+      image: janedoe/awesomeapp:v1
```

Comment:
So that's a private registry on dockerhub, I presume? You didn't want to show an example of using a privately run registry? It would be nice to be able to resolve #499.